modules/tinymce/src/core/test/ts/browser/textpatterns/ReplacementTest.ts

```diff
@@ -150,4 +152,38 @@ describe('browser.tinymce.core.textpatterns.ReplacementTest', () => {
     // This function throws an error here due to the bug identified in TINY-8779
     TinyContentActions.keystroke(editor, Keys.enter());
   });
+
+  context('TINY-8779: Matches text nodes in a paragraph', () => {
```

Comment:
JIRA numbers shouldn't be on a context/describe, it should only be used in `it` blocks. The reason being that in future we might add another test case here and then the Jira number in the context wouldn't make sense.